tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "allowImportingTsExtensions": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
+    "noUnusedParameters": false,
```

Comment:
Can we keep this on? There shouldn't be a reason to have a function with an unused parameter.

Reply:
Yeah for sure I thought I removed changes to this file from the commit...